### DISCUSSION TITLE
Problemas al intentar conectar un dispositivo bluetooh (no deja clickear mi auricular para conectarlo)

### DISCUSSION THREAD

**Discussion by sxfoazn** [blueman-project/blueman#2207] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

**Comment by infirit** [comment]
Don't speak Spanish, but usually you need to set the device in pairing mode for it to show up. Perhaps your distribution forums are a better place to ask, maybe they have a Spanish one.
